html/a7_browse_asset.html

```diff
@@ -104,22 +163,29 @@
 </script>
 
 <script>
-  function enableSwaggerUI () {
-    injectStylesheet('https://unpkg.com/swagger-ui-dist@3.12.1/swagger-ui.css');
-    injectScripts([
-      'https://unpkg.com/swagger-ui-dist@3.12.1/swagger-ui-standalone-preset.js',
-      'https://unpkg.com/swagger-ui-dist@3.12.1/swagger-ui-bundle.js'
-    ], function () {
-      var swaggerui = document.querySelector('#swagger-ui') || document.createElement('div');
-      swaggerui.id = 'swagger-ui';
-      document.querySelector('#content').prepend(swaggerui);
-
-      var ui = SwaggerUIBundle({
-        url: pathname,
-        dom_id: '#swagger-ui',
-        deepLinking: true
-      });
-      window.ui = ui;
-    }, 300);
+  try {
+    var jsonDocument = JSON.parse(content);
```

Comment:
What happens if "content" is not a json document? Should we somehow handle errors here?

Reply:
As this script module is executed systematically for all assets, it is run in a tentatively-only manner, globally.

If not json, if a primary key is missing, if anything is failing, the module fails silently (only with a console warning).

Reply:
In this case, I suggest narrowing the exception catching (smaller block) to avoid hiding other errors